src/frontends/tensorflow/src/op/block_lstm.cpp

```diff
@@ -71,7 +71,7 @@ void create_decomposed_block_lstm(const Output<Node>& x,
     auto squeeze_axis = std::make_shared<v0::Constant>(element::i32, Shape{1}, 0);
     auto xi = std::make_shared<v0::Squeeze>(xi_param, squeeze_axis);
 
-    auto lstm_cell = std::make_shared<v0::LSTMCell>(xi,
+    auto lstm_cell = std::make_shared<v4::LSTMCell>(xi,
```

Comment:
as far as I remember, you have faced an issue with v0 version of operation?
could you create a ticket for Core component? maybe we have to deprecate and delete v0 version, if it doesn't work anyway

Reply:
CVS-145457
it was talked about LSTMSequence